### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by bwignall**
Should be non-semantic.

Uses https://en.wikipedia.org/wiki/Wikipedia:Lists_of_common_misspellings/For_machines to find likely typos, with https://github.com/bwignall/typochecker to help automate the checking.

### CODE REVIEW

File: csharp/src/Google.Protobuf/ByteString.cs [hunk 1]
```diff
@@ -1,412 +1,412 @@
...
-
-        /// <summary>
-        /// Writes the entire byte array to the provided stream
-        /// </summary>
-        public void WriteTo(Stream outputStream)
-        {
-            outputStream.Write(bytes, 0, bytes.Length);
-        }
-    }
+#region Copyright notice and license
```
Comment (rafi-kamal) [inline csharp/src/Google.Protobuf/ByteString.cs]: Can you please revert the changes here?

File: csharp/src/Google.Protobuf/CodedInputStream.cs [hunk 2]
```diff
@@ -1,1657 +1,1657 @@
...
-                        throw InvalidProtocolBufferException.TruncatedMessage();
-                    }
-                    amountToSkip -= bytesRead;
-                }
-            }
-        }
-#endregion
-    }
-}
+#region Copyright notice and license
```
Comment (rafi-kamal) [inline csharp/src/Google.Protobuf/CodedInputStream.cs]: Also here

**Comment by bwignall** [comment]
@rafi-kamal , thank you for your review. I have put in the changes you requested:

I have pushed `git checkout 6263268b8c1b78a8a9b65acd6f5dd5c04dd9b0e1  -- csharp/src/Google.Protobuf/ByteString.cs csharp/src/Google.Protobuf/CodedInputStream.cs`, where I used the most recent hash from before my changes in my fork.

**Comment by rafi-kamal** [comment]
Thanks! Can you please also rebase your changes on top of master and resolve the merge conflict in protoc-artifacts/README.md?

**Comment by bwignall** [comment]
@rafi-kamal : this should be done now.
